pkg/minikube/notify/notify_test.go

```diff
@@ -34,11 +35,17 @@ import (
 	"k8s.io/minikube/pkg/version"
 )
 
+func TestMaybePrintUpdateTextFromGithub(t *testing.T) {
+	if MaybePrintUpdateTextFromGithub() {
+		t.Fatal("MaybePrintUpdateTextFromGithub() expected to return false for basic setup, bot got true")
+	}
+}
+
 func TestShouldCheckURL(t *testing.T) {
 	tempDir := tests.MakeTempDir()
 	defer os.RemoveAll(tempDir)
 
-	lastUpdateCheckFilePath := filepath.Join(tempDir, "last_update_check")
+	lastUpdateCheckFilePath := path.Join(tempDir, "last_update_check")
```

Comment:
this PR looks good after this line is changed back to  `ilepath.Join`

Reply:
fixed. Thx!